docset/winserver2012-ps/msmq/Send-MsmqQueue.md

```diff
@@ -38,19 +38,19 @@ It returns a **System.Messaging.Message** object that represents the message sen
 
 ## EXAMPLES
 
-### Example 1: Send a test message to a queue
+### Example 1: Send a test message to a discovered local queue
```

Comment:
Hello @ramonsmits Ramon, thanks for your contribution.
Is it absolutely necessary the use of the word _discovered_? Can we just say _to a local queue_?

Reply:
Well, its that the original sample uses `Get-MsmqQueue -Name "order_queue"` as there is actually no need to use this which is why I used the word discovered but I'm fine with only using local.
```suggestion
### Example 1: Send a test message to a local queue
```